tests/failpoints/cases/test_replica_stale_read.rs

```diff
@@ -265,23 +259,11 @@ fn test_update_apply_index_before_sync_read_state() {
 // item to other replicas
 #[test]
 fn test_update_resoved_ts_before_apply_index() {
-    let mut cluster = new_server_cluster(0, 3);
-    let pd_client = Arc::clone(&cluster.pd_client);
-    pd_client.disable_default_operator();
-
-    cluster.run();
-    cluster.sim.wl().start_resolved_ts_worker();
-
-    cluster.must_transfer_leader(1, new_peer(1, 1));
-    let mut leader_client = PeerClient::new(&cluster, 1, new_peer(1, 1));
+    let (mut cluster, pd_client, mut leader_client) = prepare_for_stale_read(new_peer(1, 1));
     let mut follower_client2 = PeerClient::new(&cluster, 1, new_peer(2, 2));
     leader_client.ctx.set_stale_read(true);
     follower_client2.ctx.set_stale_read(true);
 
-    // There should be no read index message while handling stale read request
-    let on_step_read_index_msg = "on_step_read_index_msg";
-    fail::cfg(on_step_read_index_msg, "panic").unwrap();
```

Comment:
why removed?

Reply:
It is not removed, it is moved to `prepare_for_stale_read` to reduce line of code.